src/components/Ladder/NewGame/NewGame.tsx

```diff
@@ -22,8 +22,8 @@ function NewGame(props: NewGameProps) : JSX.Element {
     const [open, setModalOpen] = React.useState<boolean>(false)
     const [winningPlayer, setWinningPlayer] = React.useState<DbPlayer>()
     const [losingPlayer, setLosingPlayer] = React.useState<DbPlayer>()
-    const [winningPlayerScore, setWinningPlayerScore] = React.useState<number>(0)
-    const [losingPlayerScore, setLosingPlayerScore] = React.useState<number>(0)
+    const [winningPlayerScore, setWinningPlayerScore] = React.useState<number | undefined>(undefined)
+    const [losingPlayerScore, setLosingPlayerScore] = React.useState<number | undefined>(undefined)
```

Comment:
This can just be left blank (e.g. without 'undefined') to support undefined as a type for this state (like the player one above)